.otel.env

```diff
@@ -0,0 +1,5 @@
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318
+OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
```

Comment:
IIUC, there is a cli flag that covers these two variables 

`--exporter_otlp_protocol http/protobuf`

And while short exports during development make sense, I don't know if it's worth using dotenv if we can get by with a single service name variable and exporter flag. What do you think?

Reply:
yep and much better thanks!